python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-ssh.j2

```diff
@@ -27,6 +33,12 @@ management ssh
    {{ ipv6_acl_cli }}
 {%         endfor %}
 {%     endif %}
+{%     for vrf in management_ssh.vrfs | arista.avd.natural_sort | selectattr("ipv6_access_group_in", "arista.avd.defined") %}
```

Comment:
not sure this is natural sort
```
site1-wan2(config-mgmt-ssh)#show active
management ssh
   ipv6 access-group ACL-SSH6 vrf defaus in
   ipv6 access-group ACL-SSH6 vrf MGMT in
   ipv6 access-group ACL-SSH-VRF6 vrf mgt in
   ipv6 access-group ACL-SSH6 vrf a in
   ipv6 access-group ACL-SSH6 vrf D in
   ipv6 access-group ACL-SSH6 vrf defauu in
   ipv6 access-group ACL-SSH6 vrf ZZ in
   ipv6 access-group ACL-SSH6 vrf z in
   ipv6 access-group ACL-SSH6 in
```

Reply:
removed sort for vrf when configuring acls.